Services/Mail/classes/Setup/class.ilMailDatabaseUpdateSteps.php

```diff
@@ -3,19 +3,15 @@
 /**
  * This file is part of ILIAS, a powerful learning management system
  * published by ILIAS open source e-Learning e.V.
- *
```

Comment:
IMO we MUST NOT change the license header. I know that `PhpStorm` automatically formats it, but this is wrong.